ads/mediaimpact.js

```diff
@@ -25,6 +25,21 @@ import {loadScript} from '../3p/3p';
 
 export function mediaimpact(global, data) {
   global.fif = false;
+  /* eslint google-camelcase/google-camelcase: 0 */
+  global.sas_loadHandler = function(f) {
+    if (f.hasAd) {
+      f.crea || (f.crea1 = {
```

Comment:
just want to make sure this is not a typo here. should it be `f.crea` or `f.crea1`?